refactor(image): expose constructor, unify size getters (#9179)
tauri-apps/tauri: .changes/image-size-refactor.md

---
"@tauri-apps/api": patch:breaking
"tauri": patch:breaking
---

> Removed `width` and `height` commands from the `Image` class, use `size` instead.

Reviewer: ```suggestion
Removed `width` and `height` methods on the JS `Image` class, use `size` instead.
```
